dp

```diff
@@ -136,6 +139,25 @@ else
   exit 1
 fi
 
+# Figure out deploy mode
+if [[ -z "$DEPLOY_MODE" ]]; then
+  DEPLOY_MODE=GIT
```

Comment:
Are you sure you want to make this the default?  It makes more sense to me to keep the current behaviour as the default, and to have to explicitly request the new "copy the whole repo" behaviour when you need it (which at the moment, is probably only for Oncallio, which is the only `dp`-deployed application with in-repository dependencies).